libs/blocks/chart/list.js

```diff
@@ -93,23 +93,27 @@ const getCarouselHtml = (data, hexcode) => {
           aria-label="Next Chart"></button>
       </div>
       <div id="listChartCarousel-items"
-        class="carousel-items"
-        aria-live="polite">
+        class="carousel-items">
         ${carouselItems}
-      </div>
+        </div>
+        <div class="carousel-sr" aria-live="polite"></div>
     </section>`;
 };
 
 const showCarouselItem = (element, index) => {
   const carouselItems = element.querySelectorAll('.carousel-item');
+  const carouselSR = element.querySelector('.carousel-sr');
```

Comment:
I don't believe "sr" is a standard abbreviation. Could you include a more readable name?

Reply:
SR for the variable or the class name? or both lol
For `.carousel-sr` it could be a site-wide class `.sr-only` https://webaim.org/techniques/css/invisiblecontent/

Reply:
I meant for both. Wanting to follow airbnb descriptive naming conventions https://github.com/airbnb/javascript#naming--descriptive

Reply:
Okay, it's a little long but `carouselScreenReader` and `.carousel-screen-reader`, I can't think of a shorter name

Reply:
"a11y" is commonly used as a shorter version of "accessibility" if you really want to shorten it but I think what you changed it to is fine